test/tests/functional/pbs_verify_log_output.py

```diff
@@ -113,31 +112,30 @@ def test_if_info_add(self):
         """
         Test for interface info presence in log files
         """
-        names = "["
-        for name in [x[0] for x in self.all_interfaces()]:
-            names += "(" + name + ")"
-        names += "]"
+        interfaceGenerator = self.all_interfaces()
+        for ifname in interfaceGenerator:
+            name = "[(" + ifname + ")]"
 
-        log_val = "[(ipv4)(ipv6)(ipv4/ipv6)] [(interface: )]" + names
+            log_val = "[( interface: )]" + name
```

Comment:
Oh, I understand now.  That is really strange that the regex acts like that.  Do you think it's worthy of a comment to why you're doing it?  This way someone doesn't try to strip the regex down in the future.